src/revlanguage/distributions/phylogenetics/tree/Dist_PhylodynamicBDP.cpp

```diff
@@ -201,26 +33,12 @@ RevBayesCore::AbstractBirthDeathProcess* Dist_PhylodynamicBDP::createDistributio
 const std::string& Dist_PhylodynamicBDP::getClassType( void )
 {
 
-    static std::string rev_type = "Dist_PhylodynamicBDP";
+    static std::string rev_type = "Dist_PhylodynamicBDSTP";
```

Comment:
Wondering if it's worth making the old name an alias to preserve backward compatibility...

Reply:
added